pages/Documents/ConversationalAI/ConversationBuilder/BestPractices/reusable-components.md

```diff
@@ -30,11 +30,11 @@ A better approach is to create two intents in the relevant domain--one intent fo
 
 For example:
 
-<img style="width:350px" src="img/ConvoBuilder/reusableYesNo2.png" alt="">
+<img style="width:350px" src="img/ConvoBuilder/reusableYesNo2.png" alt="An intent named Affirmation with training phrases for ok, yeah, yep, and so on">
```

Comment:
Also use–mention—quoting terms here would be clearer: ‘…named “Affirmation”… “ok,” “yeah,” …’